### ISSUE TITLE
chore: Install missing ESLint plugins

### ISSUE THREAD

**Issue by Jay-klmnop** [Jay-klmnop/Jay-C-Cloning-13#41]
## Description

This task is to resolve the configuration errors in the ESLint server by installing the missing ESLint plugins (`eslint-plugin-react`, `eslint-plugin-react-hooks`, `eslint-plugin-jsx-a11y`)
